ArtifactVersion.props

```diff
@@ -0,0 +1,7 @@
+<Project>
```

Comment:
can you please follow versioning file names/approach we are using in other repos. Like this: https://github.com/Microsoft/ApplicationInsights-dotnet/blob/develop/GlobalStaticVersion.props